### DISCUSSION TITLE
Working with Rails Live Reload

### DISCUSSION THREAD

**Discussion by budu** [julienbourdeau/debugbar#42] [Q&A]
Do anybody know of a way to make this work with Rails Live Reload?

The problem here is that when the page is auto-reloaded by Rails Live Reload, the debug just show "No request yet". Reloading manually works. Any pointer as to how to improve this?

BTW, this gem is awesome!

**Comment by julienbourdeau** [comment]
I have never used Rails Live Reload myself but I'll have a look. 
Is this the gem you're using? https://github.com/railsjazz/rails_live_reload

**Comment by budu** [comment]
Yes, this is the one.
